lib/boom_notifier/error_storage.ex

```diff
@@ -34,7 +34,7 @@ defmodule BoomNotifier.ErrorStorage do
   """
   @spec store_error(ErrorInfo.t()) :: :ok
   def store_error(error_info) do
-    error_hash_key = generate_error_key(error_info)
+    %{key: error_hash_key} = ErrorInfo.ensure_key(error_info)
```

Comment:
Shoudn't the `error_info` have a `key` already since it's added in the `ErrorInfo.build/1` function?
Same for the other places where we are calling the `ErrorInfo.ensure_key/1` function

Reply:
You mean for the errors that might be present in the storage? If that's the case, I think the user will have to stop the application to make the upgrade and all the errors will be deleted so we don't need to worry about that